draft-packages/empty-state/KaizenDraft/EmptyState/EmptyState.tsx

```diff
@@ -52,8 +52,10 @@ const EmptyState: EmptyState = ({
   children,
   straightCorners,
   useZenStyles,
+  isAnimated = true,
+  loop = false,
 }) => {
-  const theme = useTheme()
+  const animationProps = isAnimated ? { isAnimated, loop } : {}
```

Comment:
This might look a bit strange at first, but it's working as expected. If the illustration is _not_ animated, then passing through a loop property (regardless of its value) does not make sense.